ci/audit.yml

```diff
@@ -5,11 +5,11 @@ image_resource:
   type: docker-image
   source:
     repository: onsdigital/dp-concourse-tools-nancy
-    tag: latest
+    tag: 1.19.4-rc.1
```

Comment:
I don't think you should be using a release candidate
I've been using 1.19.2 elsewhere, which is good enough

Reply:
The docker image is badly named, the docker image contains Go version 1.19.4 and a version of nancy; I'll update tag today; same goes for other yml files in ci folder

Reply:
Also Go version 1.19.4 is stable Go version with security fixes; better to use latest (stable) version - https://go.dev/doc/devel/release#go1.19.minor

Reply:
its the '-rc.1' part that always rings alarm bells ...